### ISSUE TITLE
description should be "An ML-like..." and not A ML-like..."

### ISSUE THREAD

**Issue by TomerShech** [amuletml/amulet#284]

**Comment by davidgarland** [comment]
Is there actually a standard on that being the case in English or something? The ambiguity in my eyes stems from the fact that if you *expand* "ML" into "Meta Language", then you're now saying "an meta language-like", which sounds equally bad.

**Comment by SquidDev** [comment]
I think "An" is preferred, as you'd _pronounce_ it "emm ell". That's what [the internet][grammar] seems to suggest at least.

I don't know, it's not like English is a consistent language :). "An ML" does sound more correct to me at least.

[grammar]: https://www.grammar.com/a-vs-an-when-to-use/

**Comment by TomerShech** [comment]
@davidgarland @SquidDev  
TL;DR: 
> Thus, the answer is "If the word following the indefinite article begins with a vowel sound, use an; if it begins with a consonant sound, use a.

The thing is you don't expand ML into "Meta Language" because **Written language is a representation of the spoken word**.
Read here: https://english.stackexchange.com/questions/1016/do-you-use-a-or-an-before-acronyms-initialisms

Because all of that I'm sure *An ML* is the correct form.

**Comment by SquidDev** [comment]
It took me far too long to realise this was the repo description, and not in a readme/on the website 🤦.

Anyway, thanks! Fixed.
